the_list.md

```diff
@@ -3,7 +3,7 @@
 Now, you are going to add the todo-list itself to the component `app-root`. Open the file `src/app/app.component.ts`. Add the list of items inside the `AppComponent` class as an array of objects for each item. At this stage each item will contains only a title:
 
 ```js
-private todoList = [
+public todoList = [
```

Comment:
rather don't use `public` at all, everything's public by default :) let's not create JAVA from javascript shall we :P :D